models/bug_report.js

```diff
@@ -29,6 +29,11 @@ bugReportSchema.pre('save', function(next) {
   next();
 });
 
+bugReportSchema.post('save', function(report) {
+  var notify = require('../helpers/notification');
+  notify('Bug Report', 'Bug', this.created_at, this.file, '/bug-reports/', this._id);
```

Comment:
You can just pass the object itself instead of passing `this.created_at`, etc. So it's `notify(type, object, endpoint)`, e.g. 
```
notify('Bug', this, `/api/v1/bug-reports/${this._id}`)
```